honeybee_grasshopper_core/src/HB Model.py

```diff
@@ -0,0 +1,69 @@
+# Honeybee: A Plugin for Environmental Analysis (GPL)
+# This file is part of Honeybee.
+#
+# Copyright (c) 2019, Ladybug Tools.
+# You should have received a copy of the GNU General Public License
+# along with Honeybee; If not, see <http://www.gnu.org/licenses/>.
+# 
+# @license GPL-3.0+ <http://spdx.org/licenses/GPL-3.0+>
+
+"""
+Create a Honeybee Model, which can be sent for simulation.
+-
+
+    Args:
+        rooms_: A list of honeybee Rooms to be added to the Model. Note that at
+            least one Room is necessary to make a simulate-able energy model.
+        faces_: A list of honeybee Faces to be added to the Model. Note that
+            faces without a parent Room are not allowed for energy models.
```

Comment:
Right now, I know Tanushree is raising an exception for this case in the energy-model-measure but I haven't yet put anything in my workflows. I am kinda thinking that I should be raising an exception in the ToIDF component, though. What makes you think a warning is better?

Reply:
Because then you have to create two different models in order to use them for both energy and daylight simulation. Think about a model with several extra faces for interior faces that will only be included in daylight model. With the current settings then this model will fail to run in an energy simulation unless the person removes the faces from the model.

Reply:
Well, for that case, I'd you shouldn't be adding interior furniture as a Honeybee Face. It should be a Shade and you should either add it to a Room as interior furniture or you should add it to the Model as orphaned shades. Otherwise, what you have created is NOT a combination Radiance/Energy model but a model that only works for Radiance.